src-plugins/itkDataImage/readers/itkDCMTKDataImageReader.cpp

```diff
@@ -357,117 +357,104 @@ bool itkDCMTKDataImageReader::readInformation (const QStringList& paths)
+        updateMetaData(medData, medMetaDataKeys::BirthDate.key(),   d->io->GetPatientDOB().c_str());
+        updateMetaData(medData, medMetaDataKeys::Gender.key(),      d->io->GetPatientSex().c_str());
+        updateMetaData(medData, medMetaDataKeys::Description.key(), d->io->GetScanOptions().c_str());
+
+        // STUDY
+        //StudyId
+        updateMetaData(medData, medMetaDataKeys::StudyDicomID.key(),     d->io->GetStudyID().c_str());
+        updateMetaData(medData, medMetaDataKeys::StudyDescription.key(), d->io->GetStudyDescription().c_str());
+        updateMetaData(medData, medMetaDataKeys::Institution.key(),      d->io->GetInstitution().c_str());
+        updateMetaData(medData, medMetaDataKeys::Referee.key(),          d->io->GetReferringPhysicianName().c_str());
+        //StudyDate
+        //StudyTime
+
+        // SERIES
+        //SeriesID
+        updateMetaData(medData, medMetaDataKeys::SeriesDicomID.key(),     d->io->GetSeriesID().c_str());
+        updateMetaData(medData, medMetaDataKeys::SeriesNumber.key(),      d->io->GetSeriesNumber().c_str());
+        updateMetaData(medData, medMetaDataKeys::Modality.key(),          d->io->GetModality().c_str());
+        updateMetaData(medData, medMetaDataKeys::Performer.key(),         d->io->GetPerformingPhysicianName().c_str());
+        updateMetaData(medData, medMetaDataKeys::Report.key(), "");
+        updateMetaData(medData, medMetaDataKeys::Protocol.key(),          d->io->GetProtocolName().c_str());
+        updateMetaData(medData, medMetaDataKeys::SeriesDescription.key(), d->io->GetSeriesDescription().c_str());
+        //SeriesDate
+        //SeriesTime
+        //SeriesThumbnail
+
+        // IMAGE
+        //SOPInstanceUID
+        updateMetaData(medData, medMetaDataKeys::Columns.key(),         d->io->GetColumns().c_str());
+        updateMetaData(medData, medMetaDataKeys::Rows.key(),            d->io->GetRows().c_str());
+        //Dimensions
+        //NumberOfDimensions
+        updateMetaData(medData, medMetaDataKeys::Orientation.key(),     d->io->GetOrientation().c_str());
+        //Origin
+        updateMetaData(medData, medMetaDataKeys::SliceThickness.key(),  d->io->GetSliceThickness().c_str());
+        //ImportationDate
+        updateMetaData(medData, medMetaDataKeys::AcquisitionDate.key(), d->io->GetAcquisitionDate().c_str());
+        //AcquisitionTime
+        updateMetaData(medData, medMetaDataKeys::Comments.key(),        d->io->GetAcquisitionComments().c_str());
+
+        QStringList filePaths;
+        for (unsigned int i=0; i<d->io->GetOrderedFileNames().size(); i++)
+        {
+            if (d->io->GetOrderedFileNames()[i] != "")
+            {
+                filePaths << d->io->GetOrderedFileNames()[i].c_str();
+            }
+        }
+        if (filePaths.count() != 0)
+        {
+            medData->addMetaData(medMetaDataKeys::FilePaths.key(), filePaths);
+        }
```

Comment:
filePaths is a specific metadata, since it can be multiple, so we don't use `updateMetaData()` function